### PR TITLE
Version 3.8.0

### PR DESCRIPTION
**Pull request by kitsonk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,31 @@
 Releases of the extension can be downloaded from
 [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=denoland.vscode-deno).
 
+### [3.8.0](https://github.com/denoland/vscode_deno/compare/3.7.0...3.8.0) / 2021.08.10
+
+- feat: add ability to set cache directory in settings (#477)
+
+  The plugin supports setting the `deno.cache` option, which allows setting a
+  specific cache directory for the Deno language server to use. This is similar
+  to the `DENO_CACHE` environment variable that can be set when invoking Deno on
```
Comment (ry) [inline CHANGELOG.md:12]: DENO_DIR?

Comment (kitsonk) [reply CHANGELOG.md:12]: 🤦

Comment (ry) [reply CHANGELOG.md]: Makes me think this setting ought to be called something like `deno.dir` or `deno.deno_dir` ...

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**